core/src/test/java/org/elasticsearch/index/get/GetResultTests.java

```diff
@@ -160,4 +213,13 @@ public static GetResult mutateGetResult(GetResult getResult) {
         }
         return Tuple.tuple(fields, expectedFields);
     }
+
+    private static BytesReference toXContentEmbedded(GetResult getResult, XContentType xContentType) throws IOException {
```

Comment:
should you rather have an anonymous ToXContent that delegates GetResult#toXContentEmbedded and call XContentHelper#toXContent instead?